components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -3141,7 +3142,7 @@ private static int getTokenPersistPoolSize () {
             }
         } catch (NumberFormatException e) {
             if (log.isDebugEnabled()) {
-                log.debug("Error while parsing OAuth Token Persistence PoolSize", e);
+                log.debug("Error while parsing OAuth Token Persistence PoolSize" + e.getMessage(), e);
```

Comment:
Having e.getMessage() with e is redundant. e will print the whole stack trace with the message. Try to add context information such as maxPoolSizeConfigValue etc.